packages/wdio-utils/src/envDetector.ts

```diff
@@ -106,6 +106,15 @@ function isMobile(capabilities: WebdriverIO.Capabilities) {
     const browserName = (capabilities.browserName || '').toLowerCase()
     const bsOptions = capabilities['bstack:options'] || {}
     const browserstackBrowserName = (bsOptions.browserName || '').toLowerCase()
+
+    /**
+     * it is _not_ a mobile session if automationName is set to start a mobile session
```

Comment:
```suggestion
     * There are cases where sessions with `appium:*` prefixed capabilities do not fully support all "native"-mobile commands.  
     * In this case the `appium:automationName` is set with something else than the 
     * `xcuitest|uiautomator2|flutter|espress|..` value. This can be a browser driver or 
     * a "wrapped" appium browser-driver. See also https://github.com/webdriverio/webdriverio/issues/13947
     * Return `isMobile:false` for those cases. There we also accepts that specific mobile browser
     * tests (like the FF one on Android) are not seen as a mobile one
```